src/main/java/Duke.java

```diff
@@ -1,10 +1,137 @@
+            if(Check.isList(userInput)){
+                //Print out the list of tasks
+                System.out.println("Here are the tasks in your list:");
+                for(int i=0;i<taskCount;i++){
+                    System.out.println((i+1) + ". " + tasks[i]);
+                }
+                //Get user input again
+                userInput = userScan.nextLine();  
+            }
+
+            //If userInput is "unmark" get the task number and unmark the task as done
+            else if(Check.isUnmark(userInput)){
+                //get the task number
+                int taskNumber = Integer.parseInt(userInput.substring(7));
+                //mark the task as done
+                tasks[taskNumber-1].markAsNotDone();
+                //print out the task that was marked as done
+                System.out.println("OK, I've marked this task as not done yet:");
+                System.out.println(tasks[taskNumber-1]);
+                //Get user input again
+                userInput = userScan.nextLine();          
+            }
+
+            //If userInput is "mark" get the task number and mark the task as done
+            else if(Check.isMark(userInput)){
+                //get the task number
+                int taskNumber = Integer.parseInt(userInput.substring(5));
+                //mark the task as done
+                tasks[taskNumber-1].markAsDone();
+                //print out the task that was marked as done
+                System.out.println("Nice! I've marked this task as done:");
+                System.out.println(tasks[taskNumber-1]);
+                //Get user input again
+                userInput = userScan.nextLine();              
+            }
+
+            //If userInput is "todo" add a todo task to the list
+            else if(Check.isTodo(userInput)){
+                //get the task name
+                String taskName = userInput.substring(5);
+                //create a todo task
+                tasks[taskCount] = new Todo(taskName);
+                //increment the task count
+                taskCount++;
+                //print out the task that was added
+                System.out.println("Got it. I've added this task:");
+                System.out.println(tasks[taskCount-1]);
+                System.out.println("Now you have " + taskCount + " tasks in the list.");
+                //Get user input again
+                userInput = userScan.nextLine();  
+            }
+
+            //If userInput is "deadline" add a deadline task to the list
+            else if(Check.isDeadline(userInput)){
+                //get the task name
+                String taskName = userInput.substring(9,userInput.indexOf("/"));
+                //get the deadline string without any "/"
+                String deadline = userInput.substring(userInput.indexOf("/")).replace("/","");
+                //create a deadline task
+                tasks[taskCount] = new Deadline(taskName,deadline);
```

Comment:
Add appropriate space.
```suggestion
                tasks[taskCount] = new Deadline(taskName, deadline);
```